src/Components/Web.JS/src/Rendering/WebRendererInteropMethods.ts

```diff
@@ -33,35 +32,26 @@ export function attachWebRendererInterop(
   }
 
   resolveFirstRendererAttached();
-  resolveRendererAttached(rendererId);
+  invokeRendererAttachedListeners(rendererId);
```

Comment:
This change is necessary because renders can now attach and detach multiple times.